client/components/fields/editor/EventRelatedPlannings/EventRelatedPlannings.tsx

```diff
@@ -32,6 +32,7 @@ interface IProps extends IEditorFieldProps {
 export class EditorFieldEventRelatedPlannings extends React.PureComponent<IProps> {
     render() {
         const {gettext} = superdeskApi.localization;
+        const isAgendaEnabled = planningApi.planning.getEditorProfile().editor.agendas.enabled;
```

Comment:
`getEditorProfile` may not be required here, can you check if `profile` is provided in IProps (see https://github.com/superdesk/superdesk-planning/blob/develop/client/interfaces.ts#L1527)

Reply:
I already checked this, but here we got event profile, and in event profile we did not get `agenda`, so that's why we fetch planning profile here.

Reply:
Understood